tests/unit/core/reducers/test_addons.js

```diff
@@ -126,13 +121,12 @@ describe(__filename, () => {
     const extension = { ...fakeAddon, type: ADDON_TYPE_EXTENSION };
     const state = addons(undefined, loadAddonResults({ addons: [extension] }));
 
+    // We no longer store the current_version from the API addon
+    delete extension.current_version;
+
     expect(state.byID[extension.id]).toEqual({
       ...extension,
```

Comment:
I'm not sure if this is any better really but it seems you could also do this: 

```javascript
expect(state.byID[extension.id]).toEqual({
   ...createInternalAddon(extension), // *** specifically this line ***
})
```

this is instead of `delete extension.current_version;`


I thought this might be a little clearer because `createInternalAddon` is what actually extracts outs the response and gets you back what you need here..

Reply:
Good suggestion! That is a great improvement to the test.